CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+Version 1.0.3
+------
+* Make AdminAPI::Schema.get_shop a class method (impacting populate and generate commands)
```

Comment:
Minor, but I think the changelog should describe things from the end user's viewpoint, rather than the implementation details.

e.g. in this case it could say something like:

> Fix a bug which causes an error in the `populate` and `generate` commands when prompting for the shop name.

Reply:
Good point, I will apply it!